index.js

```diff
@@ -749,7 +749,16 @@ const runTests = async () => {
     .then(onTestsFinished, onTestsError)
 }
 
+// Summary is not available for GitHub Enterprise at the moment
+const isSummaryEnabled = () => {
+  return process.env.SUMMARY_ENV_VAR !== undefined
```

Comment:
Do you have a link to where this env var is documented?

Reply:
thanks for the review. The variable was not correct, I'm sorry for that. I should have imported the constant instead.
Can you please have a look at the updated code?